### PR TITLE
Add Payflex icon

### PR DESCRIPTION
**Pull request by larouxn**
Add Payflex icon, which was originally proposed by @hemabadgali in https://github.com/activemerchant/payment_icons/pull/414.

@hemabadgali, please check the icon and let me know if it's alright. I copied it from your PR.

🎩 🎩 🎩 🎩 🎩 
Screen Shot 2021-02-22 at 11 25 24
Screen Shot 2021-02-22 at 11 25 42

** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

Closes https://github.com/activemerchant/payment_icons/pull/414 (supersedes)

### CODE REVIEW

File: app/assets/images/payment_icons/payflex.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....73 1.73 0 01.59.58 2.2 2.2 0 01.31.78 3.29 3.29 0 01.07.81H29a1.49 1.49 0 00.06.52 1.21 1.21 0 00.65.75 1.57 1.57 0 00.6.11 1.19 1.19 0 00.71-.2 1 1 0 00.37-.62h.69a1.7 1.7 0 01-.61 1.06"/><path class="cls-3" d="M6.53 11a4.85 4.85 0 000 2A4.85 4.85 0 019 14.29a3.28 3.28 0 010-4.66A4.93 4.93 0 016.53 11"/><path class="cls-3" d="M4.57 11a4.59 4.59 0 001 .1 4.73 4.73 0 001-.1 4.87 4.87 0 011.3-2.53 3.3 3.3 0 01-4.65 0A5 5 0 014.57 11"/><path class="cls-4" d="M6.53 12.94a4.73 4.73 0 00-1-.1 4.59 4.59 0 00-1 .1 5 5 0 01-1.35 2.51 3.3 3.3 0 014.65 0 4.87 4.87 0 01-1.34-2.51"/><path class="cls-4" d="M4.57 12.94a5.33 5.33 0 00.09-1 5.33 5.33 0 00-.09-1 5 5 0 01-2.52-1.31A3.26 3.26 0 013 12a3.28 3.28 0 01-1 2.33 4.88 4.88 0 012.52-1.35"/><path class="cls-5" d="M2.05 8.47a.82.82 0 000 1.16A5 5 0 004.57 11a5 5 0 00-1.35-2.53.83.83 0 00-1.17 0"/><path class="cls-2" d="M2.05 15.45a.83.83 0 001.17 0 5 5 0 001.35-2.51 4.88 4.88 0 00-2.52 1.35.82.82 0 000 1.16"/><path class="cls-5" d="M9 15.45a.83.83 0 01-1.17 0 4.87 4.87 0 01-1.34-2.51A4.85 4.85 0 019 14.29a.82.82 0 010 1.16"/><path class="cls-1" d="M9 8.47a.82.82 0 010 1.16A4.93 4.93 0 016.53 11a4.87 4.87 0 011.34-2.53.83.83 0 011.13 0"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/payflex.svg:1]: - We're missing a white background. At the moment, it is transparent
- We're missing the grey border.
- Can we remove all classes and convert all `<styles>` into attributes? [alt](https://screenshot.click/17-10-rbpix-794xc.png)

**Review by tauthomas01 — COMMENTED** [review]
Thank you for creating the PR!

I left comments regarding how it looks in order to follow the guidelines.

File: app/assets/images/payment_icons/payflex.svg (hunk 1 above)
Comment (larouxn) [reply app/assets/images/payment_icons/payflex.svg]: Thanks for the heads up Thomas. I've updated the SVG accordingly. Should be good to go. :)

**Comment by hemabadgali** [comment]
@larouxn Icon looks fine.Thanks

**Review by tauthomas01 — APPROVED** [review]

**Pull request merged**